ci/test.sh

```diff
@@ -4,22 +4,33 @@ set -eux
 
 # Is there a standard interface name?
 for iface in eth0 ens4 enp0s3; do
-    IP=$(ifconfig $iface | grep 'inet addr' | cut -d: -f2 | awk '{print $1}');
+    IP=$(/sbin/ifconfig $iface | grep 'inet addr' | cut -d: -f2 | awk '{print $1}');
     if [ -n "$IP" ]; then
         echo "IP: $IP"
         break
     fi
 done
 if [ -z "$IP" ]; then
     echo "Failed to get IP, current interfaces:"
-    ifconfig -a
+    /sbin/ifconfig -a
```

Comment:
was `/sbin/` not on PATH already? I think it had to be in order to be working.

Reply:
it didn't seem to work in the local environment created by Vagrantfile